src/qt/guiutil.cpp

```diff
@@ -246,8 +246,8 @@ void openDebugLogfile()
 #endif
 }
 
-ToolTipToRichTextFilter::ToolTipToRichTextFilter(int size_threshold, QObject *parent):
-    size_threshold(size_threshold), QObject(parent)
+ToolTipToRichTextFilter::ToolTipToRichTextFilter(int size_threshold, QObject *parent) :
```

Comment:
Ahh right. I didn't realise this was part of this commit. Thanks for merging the other pull.